README.md

```diff
@@ -33,24 +30,25 @@ These are marked as `peerDependencies` of `stryker-karma-runner` so you get a wa
 
 ## Configuring
 
-You can either configure the karma test runner from the `stryker.conf.js` file or from the command line. This readme describes how to do it via the config file.
+You can either configure the karma test runner from the `stryker.conf.js` file or from the command line. 

```

Comment:
You can either configure the karma test runner using the command line or by providing a the `stryker.conf.js` file.